joe/joe.py

```diff
@@ -61,13 +69,23 @@ def _handle_gitignores(names):
                    'files.') % name
             exit = True
             break
-        output += '\n#####=== %s ===#####\n' % raw_name
-        filepath = os.path.join(DATA_DIR, raw_name + '.gitignore')
-        output += '\n'
-        with open(filepath) as f:
-            output += f.read()
-    if not exit:
-        print output
+        _fetch_gitignore(raw_name)
+
+
+def _fetch_gitignore(raw_name, directory=''):
```

Comment:
Looks good. Would be helpful to include this same note in the docs of this method.

```
'''one-line description of method.

longer desc spanning
multiple lines explaining what happens here.
'''
```